spec/ddtrace/contrib/grpc/datadog_interceptor/client_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'spec_helper'
+require 'grpc'
+require 'ddtrace'
+
+RSpec.describe 'tracing on the client connection' do
+  subject { Datadog::Contrib::GRPC::DatadogInterceptor::Client.new }
```

Comment:
I suggest giving `subject` blocks names. Something like `subject(:client)` would be good here.